src/main/java/com/squareup/catalog/demo/example/CreateItemExample.java

```diff
@@ -69,40 +69,38 @@ private CatalogObject createItem(CatalogApi catalogApi) throws IOException {
      *
      * Note: this call only *creates* the new objects and packages them for
      * upsert. Nothing has been uploaded to the server at this point.
-     **/
-    BatchUpsertCatalogObjectsRequest request = new BatchUpsertCatalogObjectsRequest.Builder()
-        .idempotency_key(UUID.randomUUID().toString())
-        .batches(singletonList(new CatalogObjectBatch.Builder()
+     */
+    BatchUpsertCatalogObjectsRequest request = new BatchUpsertCatalogObjectsRequest()
+        .idempotencyKey(UUID.randomUUID().toString())
+        .batches(singletonList(new CatalogObjectBatch()
             .objects(singletonList(
                 item("#SODA", "Soda", null,
                     itemVariation("#SODA-SMALL", "Small", 150),
                     itemVariation("#SODA-MEDIUM", "Medium", 175),
                     itemVariation("#SODA-LARGE", "Large", 200)
-                )))
-            .build()))
-        .build();
+                )))));
 
-    /**
+    /*
      * Post the batch upsert to insert the new item.
      *
      * Use the BatchUpsertCatalogObjectsRequest object we just created to
      * upsert the new CatalogObjects to the catalog associated with the
      * access token included on the command line.
-     **/
+     */
     logger.info("Creating new Soda item");
-    BatchUpsertCatalogObjectsResponse response = catalogApi.batchUpsert(request);
+    BatchUpsertCatalogObjectsResponse response = catalogApi.batchUpsertCatalogObjects(request);
 
     // If the response is null, we already logged errors.
     if (response == null) {
```

Comment:
`response == null` is ever true. Either an `APIException` is thrown or a non-null `BatchUpsertCatalogObjectsResponse` is returned.